doxygen/15_UserGuide.dox

```diff
@@ -167,7 +167,19 @@ GeNN has no explicitly hard-coded synapse and neuron variables. Users are free t
 - \c DT : Time step (typically in ms) for simulation;
 Neuron integration can be done in multiple sub-steps inside the neuron model for numerical stability (see Traub-Miles and Izhikevich neuron model variations in \ref sectNeuronModels).
 
-- \c inSyn: This is an intermediary synapse variable which contains the summed input into a postsynaptic neuron (originating from the \$(addToInSyn, X) or \$(addToInSynDelay, X, Y) functions of the weight update model used by incoming synapses) . 
+\note DT exceptionally does not require bracketing with $(.)
+
+- \c t : The current time as a floating point value
+
+- \c id : The index of the neuron/ synapse in the population
```

Comment:
``id `` is only valid in the neuron context so in neurons, current sources, postsynaptic models; and pre and postsynaptic weight update models. ``id_syn`` gives some sort of index of the current synapse.